unsupported/Eigen/CXX11/src/Tensor/TensorContractionGpu.h

```diff
@@ -1213,11 +1071,6 @@ EigenFloatContractionKernelInternal(const LhsMapper lhs, const RhsMapper rhs,
     __syncthreads();
   } // end loop over k
 
-#if defined(EIGEN_HIPCC)
-#undef prefetch_lhs_hipcc
-#undef prefetch_rhs_hipcc
```

Comment:
@deven-amd in this PR you removed HIP-specific `prefetch_lhs_hipcc` and `prefetch_rhs_hipcc`. Do we need a special version of HIP to make this work, or the released version in ROCm 1.8.1 work just fine?

Reply:
I have been using the same version of HIP that we use for TF (`roc-1.8.x` with the PR 457 fix) for all of my eigen testing. As long as we use that we should be fine (assuming that version is a pre-req for TF too).

Also these HIP-specific macros were needed only because we had not enabled the Eigen "packet" code for HIP. Since this change enables for Eigen "packet" code for HIP, those macros are no longer needed.